src/Symfony/Component/Process/Process.php

```diff
@@ -1174,6 +1175,20 @@ public function setEnv(array $env)
      * @return resource|string|\Iterator|null The Process input
      */
     public function getInput()
+    {
+        @trigger_error(sprintf('The %s::getInput will return the given input in Symfony 5.0, use %s::getValidatedInput() instead.', __CLASS__, __CLASS__), E_USER_DEPRECATED);
+
+        return $this->input;
+    }
+
+    public function getOriginalInput()
+    {
+        @trigger_error(sprintf('The %s::getOriginalInput is going to be deleted since %s::getInput() will return the original input in Symfony 5.0.', __CLASS__, __CLASS__), E_USER_DEPRECATED);
```

Comment:
Introducing a new method and deprecating it at the same time makes no sense as it means nobody should ever call it.